pkg/scheduler/util/scheduler_helper.go

```diff
@@ -75,13 +75,13 @@ func PrioritizeNodes(task *api.TaskInfo, nodes []*api.NodeInfo, fn api.NodeOrder
 }
 
 // SelectBestNode returns nodes by order of score
-func SelectBestNode(nodeScores map[int][]*api.NodeInfo) []*api.NodeInfo {
+func SelectBestNode(nodeScores map[float64][]*api.NodeInfo) []*api.NodeInfo {
```

Comment:
hm... it's better to work with https://github.com/kubernetes-sigs/kube-batch/issues/699 :)

/cc @hex108

Reply:
ok, I'll send PR after this.